apps/google-analytics-4/src/locations/ConfigScreen.tsx

```diff
@@ -1,68 +1,135 @@
-import React, { useCallback, useState, useEffect } from 'react';
+import { useCallback, useState, useEffect } from 'react';
 import { AppExtensionSDK } from '@contentful/app-sdk';
-import { Heading, Form, Paragraph, Flex } from '@contentful/f36-components';
+import { Heading, Form, Paragraph } from '@contentful/f36-components';
 import { css } from 'emotion';
 import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
```

Comment:
Another comment-cleanup here

Reply:
Ah you mean the `/* useCMA */` bit. Good catch. Will remove on the next PR.